.github/workflows/release.yaml

```diff
@@ -0,0 +1,99 @@
+name: Release
+
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+"
+jobs:
+  main:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        go: ["1.16"]
+    name: Go ${{ matrix.go }}
+    steps:
+      - name: Checkout Metal LB Operator
+        uses: actions/checkout@v2
+        with:
+          path: metallboperator
+          fetch-depth: 0 # Fetch all history for all tags and branches
+
+      - uses: actions/setup-go@v2
+        id: go
+        with:
+          go-version: ${{ matrix.go }}
+
+      - name: Verify modules
+        run: go mod verify
+
+      - name: Verify format
+        run: |
+          make fmt
+          git diff --exit-code
+
+      - name: Verify bundle manifests
+        run: |
+          make bundle
+          git diff --exit-code
+
+      - name: Create and set up K8s Kind Cluster
+        run: |
+          ./scripts/kind-cluster-with-registry.sh
+          make deploy-olm
+
+      - name: Build bundle image
+        run: |
+          make build-and-push-bundle-images REPO=localhost:5000
+
+      - name: Deploy Metal LB Operator with OLM
+        run: |
+          make deploy-with-olm REPO=localhost:5000
```

Comment:
Please add help to this target its not showing in make help

Reply:
I'll file a separate PR for that, doesn't make much sense in this context.